client/src/app/gateways/repositories/motions/motion-repository.service/motion-repository.service.ts

```diff
@@ -263,8 +263,8 @@ export class MotionRepositoryService extends BaseAgendaItemAndListOfSpeakersCont
     }
 
     public override getFieldsets(): Fieldsets<Motion> {
-        const routingFields: TypedFieldset<Motion> = [`sequential_number`, `meeting_id`];
-        const titleFields: TypedFieldset<Motion> = routingFields.concat([`title`, `number`, `created`, `forwarded`]);
+        const routingFields: TypedFieldset<Motion> = [`sequential_number`, `meeting_id`, `title`];
+        const titleFields: TypedFieldset<Motion> = routingFields.concat([`number`, `created`, `forwarded`]);
```

Comment:
Is this really necessary? Title is not required for routing and therefore should not be included in routing fields.